app/assets/stylesheets/layout/common/_footer.scss

```diff
@@ -38,19 +62,161 @@
+  @mixin counter-inline() {
+    text-align: left;
+
+    .button {
+      position: absolute;
+    }
+
+    .social-sharing__counter {
+      position: relative;
+      top: 27.5px;
+      transform: translateY(-50%);
+      margin: 0 0 0 55px;
+      padding: 0 0 0 10px;
+    }
+
+    .footer-primary__list-item {
+      position: absolute;
+      top: 50%;
+      margin-top: -27.5px;
+    }
+  }
+
+  .button,
+  .button:active,
+  .button:focus {
+    border-bottom: none;
+    border-radius: 50%;
+    border: none;
+  }
+
+  .social-sharing__counter {
+    margin: $baseline-unit 0;
+    color: $color-black;
+  }
+
+  margin-top: 0;
+  padding-left: 0;
+  text-align: center;
+
+  @include respond-to($mq-s, $mq-s-max) {
+    @include counter-inline;
+  }
+
+  @include respond-to($mq-xl) {
+    @include counter-inline;
+  }
+
+  .twitter-link,
+  .twitter-link:active,
+  .twitter-link:focus {
+    background: $color-twitter;
+    padding: 18.5px 16.426px; // (55px - 18px) / 2 - (55px - (184.573px * (18 / 150))) / 2
+
+    &:hover {
+      background: $color-twitter-hover;
+    }
+  }
+
+  .facebook-link,
+  .facebook-link:active,
+  .facebook-link:focus {
+    background: $color-facebook;
+    padding: 16.5px 21.788px; // (55px - 22px) / 2 - (55px - (77.8924px * (22 / 150))) / 2
+
+    &:hover {
+      background: $color-facebook-hover;
+    }
+  }
+
+  .youtube-link,
+  .youtube-link:active,
+  .youtube-link:focus {
+    background: $color-youtube;
+    padding: 18.5px 14.704px; // (55px - 18px) / 2 - (55px - (213.264px * (18 / 150))) / 2
+
+    &:hover {
+      background: $color-youtube-hover;
+    }
+  }
+}
+
+.social-sharing__counter__total {
+  font-weight: bold;
+  display: block;
+  @include body(24, 24);
+
+  @include respond-to($mq-l) {
+    @include body(30, 30);
+  }
+}
+
+.social-sharing__counter__label {
+  display: block;
+  // font-size: 0.875rem;
+  @include body(14, 14);
+}
+
 .l-footer-primary__list--left {
-  @include column(6);
+  @include column(4);
+  min-height: 55px;
+
+  @include respond-to($mq-s, $mq-s-max) {
+    @include column(12);
+    position: relative;
+  }
+}
 
-  @include respond-to($mq-s, 37.5em) {
+.l-footer-primary__list--centre {
+  @include column(4);
+  min-height: 55px;
+
+  @include respond-to($mq-s, $mq-s-max) {
     @include column(12);
+    position: relative;
   }
 }
 
 .l-footer-primary__list--right {
-  @include column(6);
+  @include column(4);
+  min-height: 55px;
 
-  @include respond-to($mq-s, 37.5em) {
+  @include respond-to($mq-s, $mq-s-max) {
     @include column(12);
+    position: relative;
+  }
+
+  .footer-primary__list-item {
+    line-height: 1;
+
+    .footer-primary__blog-link,
+    .footer-primary__blog-link:visited,
+    .footer-primary__blog-link:focus,
+    .footer-primary__blog-link:active {
+      background: $color-mas-blog-blue;
+      border-radius: 5px;
+      border-bottom: none;
+      color: white;
+      font-weight: 700;
+      padding: $baseline-unit*2 1.5rem;
```

Comment:
Probably a good candidate for the `px` function here actually with the horizontal padding